src/dlgProfilePreferences.cpp

```diff
@@ -63,6 +69,105 @@ dlgProfilePreferences::dlgProfilePreferences(QWidget* pF, Host* pH) : QDialog(pF
     // /testing controls can be placed if needed...
     groupBox_Debug->hide();
 
+    // The presence of an entry here does not mean that we will actually produce
+    // or maintain a particular language - what choices get made available to
+    // the user gets determined later, by the presence of the "compiled" binary
+    // translation (".qm") file, NOT here.  This just records the POSSIBLE
+    // choices that someone had allowed for, more are welcome...
+    // A "none" entry should always be present - it is a dummy entry that has
+    // NO translation file (and unloads any translation files) and is inserted
+    // further down!
+
+// LANGUAGE: 1 - enter code and native description string:
+    //                language file mudlet_     .qm    NATIVE name for language:
+    mTranslationMap.insert(QStringLiteral("en_US"), QStringLiteral("English (American)"));
+    mTranslationMap.insert(QStringLiteral("en_GB"), QStringLiteral("English (British)"));
+    QStringList tooltipLanguageEntries;
+
+    // This covers the default en_US as well as en_GB:
+    tooltipLanguageEntries.append(QStringLiteral("Choose the language for Mudlet to use..."));
```

Comment:
Should be `Choose the language for Mudlet to use:`

elipsis is used to indicate that there is more following in another menu in UI design.

Reply:
Well, I meant the [ellipsis](https://en.wikipedia.org/wiki/Ellipsis#On_the_Internet_and_in_text_messaging) to mean if you select this control it carries out this action`...`

Grammatically I might not be entirely correct, but I suspect that a colon `:` is definitely not right here.